docs/metrics/getting-started-prometheus-grafana/README.md

```diff
@@ -1,12 +1,14 @@
 # Getting Started with Prometheus and Grafana
 
-- [Export metrics from the application](#export-metrics-from-the-application)
-  - [Check results in the browser](#check-results-in-the-browser)
-- [Collect metrics using Prometheus](#collect-metrics-using-prometheus)
-  - [Configuration](#configuration)
-  - [Start Prometheus](#start-prometheus)
-  - [View results in Prometheus](#view-results-in-prometheus)
-- [Explore metrics using Grafana](#explore-metrics-using-grafana)
+- [Getting Started with Prometheus and Grafana](#getting-started-with-prometheus-and-grafana)
```

Comment:
self-referencing link?

Reply:
I'll remove this, the VSCode markdown file plugin did this. :grin: